api/services/ocean/spark/schemas/oceanSparkClusterConfigIngress.yaml

```
@@ -0,0 +1,12 @@
+type: object
+description: >
+  The object specifying the Ocean Spark Ingress configuration.
+properties:
+  serviceAnnotations:
+    type: object
+    additionalProperties:
+      type: string
+    description: |
+      Extra annotations to the ingress load balancer service
+    example:
+      securityGroupId: sg-03ba056e31f183482
```

Here's a real-world example to set the security group 
```suggestion
      service.beta.kubernetes.io/aws-load-balancer-security-groups: sg-03ba056e31f183482
```